400-claims.md

```diff
@@ -92,7 +98,7 @@ Source: [400.01-claim.json](examples/400.01-claim.json)
     - underway: in progress. This should only be used if the invocation container MUST exit before it can determine whether all operations are complete. Note that `underway` is a _long term status_ that indicates that the installation's final state cannot be determined by the system. For this reason, it should be avoided.
     - unknown: the state is unknown. This is an error condition.
     - success: completed successfully
-- revision: An [ULID](https://github.com/ulid/spec) that MUST change each time the release is modified.
+- revision: An [ULID](https://github.com/ulid/spec) that MUST change each time the installation is modified. It MUST NOT change when a read-only operation is performed on an installation.
```

Comment:
It might be useful to clarify that read-only means when an action with the attribute `"modifies": false` is performed?

https://github.com/cnabio/cnab-spec/blob/master/101-bundle-json.md#custom-actions